src/time.cr

```diff
@@ -53,21 +53,55 @@ require "crystal/system/time"
 # span.hours # => 1
 # ```
 struct Time
-  # *Heavily* inspired by Mono's DateTime class:
-  # https://github.com/mono/mono/blob/master/mcs/class/corlib/System/DateTime.cs
-
   include Comparable(self)
 
-  TicksMask       =      0x3fffffffffffffff
-  KindMask        =      0xc000000000000000
-  MAX_VALUE_TICKS = 3155378975999999999_i64
+  # :nodoc:
+  DAYS_MONTH = [0, 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
 
-  DAYS_MONTH      = [0, 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
+  # :nodoc:
   DAYS_MONTH_LEAP = [0, 31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
 
-  DP400 = 146097
-  DP100 =  36524
-  DP4   =   1461
+  # :nodoc:
+  SECONDS_PER_MINUTE = 60
```

Comment:
Why shouldn't this be documented? It's a "natural" constant and might be useful for special time-related code, so it I'd like to see them in the API docs.

Reply:
No, they are part of the internal implementation. If the implementation changes I don't want to have to keep this constant. Anyone can compute this by themselves.